docs/querying/sql-functions.md

```diff
@@ -1516,18 +1603,24 @@ COUNT DISTINCT is an alias for [`APPROX_COUNT_DISTINCT`](#approx_count_distinct)
 
 <details><summary>Example</summary>
 
-The following example counts the number of distinct airlines reported in `flight-carriers`:
+The following example counts the number of distinct flights per day after `'2005-01-01 00:00:00'` in `flight-carriers`:
 
 ```sql
-SELECT COUNT(DISTINCT "Reporting_Airline") AS "num_airlines"
+SELECT
+  TIME_FLOOR(__time, 'P1D') AS "flight_day",
+  COUNT(*) AS "num_flights"
 FROM "flight-carriers"
+GROUP BY 1
```

Comment:
```suggestion
WHERE __time > '2005-01-01 00:00:00'
GROUP BY 1
```
think this one is missing the where clause we talked about (and described in line 1606)